### PR TITLE
[ Do NOT MERGE ] Remove unnecessary code

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -176,13 +171,9 @@ function xpathiOS(name) {
...
 function createIosElement(element) {
-    let xCUIElementType = "XCUIElementType";
-    let uIA = "UIA";
-    let elementType;
-    if (capability.platformVersion.includes("10")) {
-        elementType = xCUIElementType;
-    } else {
-        elementType = uIA;
+    let elementType = "UIA";
+    if (caps.platformVersion.includes("10")) {
```
Comment (zbranzov) [inline index.js]: Not sure if this will work as we don't have 'caps' in outer scope

File: index.js [hunk 2]
```diff
@@ -86,20 +81,20 @@ function log(message) {
     }
 }
 
-exports.configureLogging = function (driver) {
-    driver.on("status", function (info) {
+exports.configureLogging = function(driver) {
+    driver.on("status", function(info) {
```
Comment (zbranzov) [inline index.js]: These lines look like difference because not formatted.

File: test-runner.js [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
-var child_process = require("child_process");
 
-var verbose = process.env.npm_config_loglevel === "verbose";
-var capabilitiesLocation = process.env.npm_config_capsLocation;
+const fs = require("fs");
+const path = require("path");
+const portastic = require("portastic");
+const child_process = require("child_process");
+const utils = require("./utils");
+let appium = "appium";
```
Comment (zbranzov) [inline test-runner.js]: const as well ?

File: postinstall.js [hunk 4]
```diff
@@ -19,24 +24,22 @@ try {
 }
 
 if (generateSampleTest) {
-    var sampleTestSrc = path.join(__dirname, "sample-test.js");
-    var sampleTestDest = path.join(testsDir, "sample-test.js");
+    let sampleTestSrc = path.join(__dirname, "sample-test.js");
```
Comment (zbranzov) [inline postinstall.js]: utils.resolve() ?

File: test-runner.js [hunk 5]
```diff
@@ -158,25 +164,13 @@ function waitForOutput(process, matcher, timeout) {
...
-    var appAppiumCapabilitiesLocation = path.join(projectDir, fileName);
-    var customCapabilitiesLocation;
-
-    if(capabilitiesLocation){
-        customCapabilitiesLocation = path.join(projectDir, capabilitiesLocation, fileName);
-    }
+    const fileName = "appium.capabilities.json";
+    const pluginAppiumCapabilitiesLocation = utils.resolve(pluginRoot, fileName);
+    const appAppiumCapabilitiesLocation = utils.resolve(projectDir, fileName);
+    const customCapabilitiesLocation = utils.resolve(projectDir, capabilitiesLocation, fileName);
```
Comment (zbranzov) [inline test-runner.js]: will this return result if no capabilitiesLocation is provided ?

File: test-runner.js (hunk 3 above)
Comment (SvetoslavTsenov) [inline test-runner.js]: nope
cause there is another check if we run tests on windows and it should be init again like 

if (process.platform === "win32") {
    appium = "appium.cmd";
    mocha = "mocha.cmd";
}

File: index.js (hunk 1 above)
Comment (SvetoslavTsenov) [inline index.js]: Still not tested with iOS, but pretty sure it won't. I will push fix later

File: test-runner.js [hunk 6]
```diff
@@ -158,25 +164,13 @@ function waitForOutput(process, matcher, timeout) {
...
-    var pluginRootDir = path.dirname(projectDir);
-    var pluginAppiumCapabilitiesLocation = path.join(pluginRootDir, fileName);
-    var appAppiumCapabilitiesLocation = path.join(projectDir, fileName);
-    var customCapabilitiesLocation;
-
-    if(capabilitiesLocation){
-        customCapabilitiesLocation = path.join(projectDir, capabilitiesLocation, fileName);
-    }
+    const fileName = "appium.capabilities.json";
+    const pluginAppiumCapabilitiesLocation = utils.resolve(pluginRoot, fileName);
```
Comment (zbranzov) [inline test-runner.js]: pluginRoot here is different from what we expect to be. It should be the the parent of 'demo' folder. This is exactly what path.dirname(projectDir) returns

File: test-runner.js (hunk 3 above)
Comment (zbranzov) [inline test-runner.js]: you are right

**Pull request merged**